docker/README.md

```diff
@@ -0,0 +1,88 @@
+# GitHub Automation App Setup with Docker
```

Comment:
Please add a section in the main README under `deployment/docker` heading, and link to this docker README for easier access.